tests/connectors/anghami.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = function(driver, connectorSpec) {
+	connectorSpec.shouldBehaveLikeMusicSite(driver, {
+		url: 'https://www.anghami.com/explore',
+	});
```

Comment:
:+1: Yeah, you're right, seems like Anghami doesn't always play a song automatically on the explore page, but if you go to a specific song url, https://www.anghami.com/song/18025331 for example, the player will pop up and it will always play the song automatically, I tried it and it passes now.